rep/db/db.py

```diff
@@ -35,6 +35,16 @@ def create_db(self):
                 FOREIGN KEY (sourceType) REFERENCES raw_vote_object_source_type(type), 
                 FOREIGN KEY (format) REFERENCES raw_vote_object_format(format)
             );
+            
+        CREATE TABLE IF NOT EXISTS processed_vote_results
+            (
+                year INTEGER NOT NULL,
+                date VARCHAR NOT NULL,
+                billNumber VARCHAR NOT NULL,
+                voteName VARCHAR NOT NULL,
+                repName VARCHAR NOT NULL,
+                repVote VARCHAR NOT NULL
```

Comment:
do we know the full set of votes that people can make? Is it just `yes/no/abstain`?

Reply:
I think so, but I'm not 100% certain